x/profiles/keeper/keeper_chain_link.go

```diff
@@ -0,0 +1,148 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	sdkquery "github.com/cosmos/cosmos-sdk/types/query"
+
+	"github.com/desmos-labs/desmos/x/profiles/types"
+)
+
+// StoreChainLink stores the given chain link inside the current context.
+// It assumes that the given chain link has already been validated.
+func (k Keeper) StoreChainLink(ctx sdk.Context, link types.ChainLink, destinationAddress string) error {
+	if _, found := k.GetChainLink(ctx, link.ChainConfig.Name, link.Address); found {
+		return fmt.Errorf("chain link already exists")
+	}
+
+	// Check if address has the profile
+	profile, found, err := k.GetProfile(ctx, destinationAddress)
+	if err != nil {
+		return err
+	}
+	if !found {
+		return fmt.Errorf("address does not have any profile")
+	}
+	// Store chain link to the profile
+	profile.ChainsLinks = append(profile.ChainsLinks, link)
+	if err := k.StoreProfile(ctx, profile); err != nil {
+		return err
+	}
+
+	store := ctx.KVStore(k.storeKey)
+	key := types.ChainsLinksStoreKey(link.ChainConfig.Name, link.Address)
+	store.Set(key, k.cdc.MustMarshalBinaryBare(&link))
+	return nil
+}
+
+// GetChainLink returns the chain link corresponding to the given address and the given chain name inside the current context.
+func (k Keeper) GetChainLink(ctx sdk.Context, chainName string, address string) (link types.ChainLink, found bool) {
+	store := ctx.KVStore((k.storeKey))
+
+	bz := store.Get(types.ChainsLinksStoreKey(chainName, address))
+	if bz == nil {
+		return types.ChainLink{}, false
+	}
+	k.cdc.MustUnmarshalBinaryBare(bz, &link)
+	return link, true
+}
+
+// GetAllChainsLinks returns a list of all the chains links inside the given context.
+func (k Keeper) GetAllChainsLinks(ctx sdk.Context) []types.ChainLink {
```

Comment:
Can you move this method to a new file named `alias_functions`? It's usually where we place all these kind of methods that use iterators inside them to implement a utility method